pages/api/awards/get.js

```diff
@@ -50,12 +50,15 @@ export const getAward = async filter => {
   // Remove the relation table data
   return {
     ...award,
-    awards_performances: award.awards_performances.map(performance => {
-      return {
-        ...performance.performances,
-        nominee_count: performance.nominee_count,
-        status: performance.status,
-      };
-    }),
+    awards_performances: award.awards_performances.map(
```

Comment:
Is there a good way to rename this field to performances? If I change to awards, the response will have awards_performances from ...award as well. Would deleting the key be the best way? Same question for performances/get.js

Reply:
I'm not too sure, we can keep all the information for now.  No extra harm right? @OustanDing

Reply:
It's fine to keep this as `awards_performances`.